test/Microsoft.ML.Tests/Scenarios/IrisPlantClassificationWithStringLabelTests.cs

```diff
@@ -30,6 +30,14 @@ public void TrainAndPredictIrisModelWithStringLabelTest()
             pipeline.Add(new StochasticDualCoordinateAscentClassifier());
 
             PredictionModel<IrisDataWithStringLabel, IrisPrediction> model = pipeline.Train<IrisDataWithStringLabel, IrisPrediction>();
+            string[] scoreLabels;
+            model.TryGetScoreLabelMapping(out scoreLabels);
+
+            Assert.NotNull(scoreLabels);
+            Assert.Equal(3, scoreLabels.Length);
+            Assert.True(scoreLabels[0] == "Iris-setosa");
+            Assert.True(scoreLabels[1] == "Iris-versicolor");
+            Assert.True(scoreLabels[2] == "Iris-virginica");
```

Comment:
I see you know about `Assert.Equal` since you used it in the line above. Maybe these three equality conditions should be `Assert.Equal`.